app/controllers/admin/domain_blocks_controller.rb

```diff
@@ -43,12 +43,8 @@ def create
     def update
       authorize :domain_block, :update?
 
-      @domain_block.update(update_params)
-
-      severity_changed = @domain_block.severity_changed?
-
-      if @domain_block.save
```

Comment:
Seems like you fixed a bug, `update` would have saved to database and `severity_changed?` would have been false.